homeassistant/components/bayesian/binary_sensor.py

```diff
@@ -21,6 +22,7 @@
 from homeassistant.helpers.event import async_track_state_change
 
 ATTR_OBSERVATIONS = "observations"
+ATTR_OBSERVED_ENTITIES = "observed_entities"
```

Comment:
This name is not clear enough that it only includes entities of which there is a current known observation instead of all entities that are being observed.

Reply:
I've now used the same verbiage of the documentation: occurred_observation_entities